app/services/variant_units/option_value_namer.rb

```diff
@@ -56,7 +56,7 @@ def option_value_value_unit
     def option_value_value_unit_scaled
       unit_scale, unit_name = scale_for_unit_value
 
-      value = BigDecimal(@variant.unit_value / unit_scale, 6)
+      value = BigDecimal(@variant.unit_value / unit_scale, 6).truncate(2)
```

Comment:
But couldn't help but try to understand this further.
I'm unsure what the `6` is doing on this line. Apparently it's the number of [significant digits](https://ruby-doc.org/stdlib-2.5.1/libdoc/bigdecimal/rdoc/BigDecimal.html#:~:text=terminate%20the%20value.-,digits,the%20number%20of%20significant%20digits%20is%20determined%20from%20the%20initial%20value.,-The%20actual%20number) for the new BigDecimal, but it's obviously not limiting the number that we're seeing on the screen. So maybe it's no longer needed.

So I'm curious to see if this refactor works. Would you like to apply and see if tests still pass?
```suggestion
      value = (@variant.unit_value / unit_scale).to_d.truncate(2)
```

Reply:
Same I am not sure what the `6` is doing here. It was added by @jibees here : https://github.com/openfoodfoundation/openfoodnetwork/pull/9497/commits/4452ad2c519a59f790985c460a2b79346a3e43f1 maybe he could provide more context.

Your suggested change works fine.

Reply:
Great, thanks for looking into that! I guess it was intended to do the same thing as `.truncate(6)`.. but it obviously didn't work.